### PR TITLE
dependencies update

### PR DESCRIPTION
**Pull request by hubertshelley**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -11,9 +11,9 @@ keywords = ["casbin", "asyncio", "access control", "Tortoise ORM", "authorizatio
 include = ["CHANGELOG.md", "casbin_tortoise_adapter/py.typed"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
 asynccasbin = "^1.1.2"
-tortoise-orm = {extras = ["accel"], version = "^0.19.1"}
+tortoise-orm = { extras = ["accel"], version = "^0.20" }
```
Comment (thearchitector) [inline pyproject.toml]: hmmmm ... dropping support for pydantic 1 **and** python 3.7 is a pretty massive breaking change

im not sure how many people are on the latter, but absolutely people are still on the former.

I'd need to be convinced to bump up, I think, knowing that people using casbin are probably dealing with pretty complex apps

Comment (hsluoyz) [reply pyproject.toml]: @hubertshelley it's not good to update dependency and drop old versions with no clear reason or benefits

Comment (thearchitector) [reply pyproject.toml]: it doesnt seem like the actual API has changed, so if its just about version targeting, i would be open to something like this: https://python-poetry.org/docs/dependency-specification/#multiple-constraints-dependencies

Comment (hubertshelley) [reply pyproject.toml]: Thank you for your review suggestion. I am not very familiar with poetry. This upgrade method is very good. I learned a lot.

**Comment by thearchitector** [comment]
@hubertshelley thanks! i had to fix the CI pipeline because my dev tools dropped 3.7 support, but looks good now!

**Comment by thearchitector** [comment]
released in 1.2.1

**Pull request merged**